packages/test/test-pairwise-generator/src/test/generatePairwiseOptions.spec.ts

```diff
@@ -108,15 +108,15 @@ function  validatePairsExhaustively<T>(
 
 describe("generatePairwiseOptions",()=>{
     it("SimpleOptions",()=>{
-        assert.strictEqual(simpleValues.length, 10, optionsToString(...simpleValues));
+        assert.strictEqual(simpleValues.length, 8, optionsToString(...simpleValues));
```

Comment:
less is more when it comes to pairwise. it means we are covering more pairs with fewer values, so we can more efficiently explore the space